MozziGuts_impl_MBED.hpp

```diff
@@ -14,9 +14,47 @@
 #  error "Wrong implementation included for this platform"
 #endif
 
+#define CHUNKSIZE 64
 
 ////// BEGIN analog input code ////////
 
+#if (USE_AUDIO_INPUT)
```

Comment:
As said, I think that, on the long run, this should probably be moved into a separate AudioConfigInput file?